replace_hash.py

```diff
@@ -0,0 +1,93 @@
+"""This scipt is used to replace specified placeholders(string) in a file(stored under ./output) with hashed random strings.
+Usage:
+      (bash) python3 replace_hash.py <filename>
+"""
+
+import re
+import random
+import string
+import sys
+from hashlib import sha256
+
+"""Read arguments in bash and store the filename in a variable.
+
+If filename not given in the bash code, program exit with error message.
+"""
+args = sys.argv
+if (len(args) < 2):
+  sys.exit("No filename given!")
+param_filename = args[1] 
```

Comment:
It is good practice to:
- Not split your module-level code. So probably move this to just before the block in lines 91-93.
- Add a condition on whether this is [executing as a main module](https://docs.python.org/3.8/library/__main__.html). So please add that there too.